janus_core/cli.py

```diff
@@ -90,6 +94,22 @@ def parse_typer_dicts(typer_dicts: list[TyperDict]) -> list[dict]:
     return typer_dicts
 
 
+def _iter_path_to_str(dictionary: dict) -> None:
```

Comment:
`iter` to me suggests this should return a generator. Probably `_dict_paths_to_strs`?